x-pack/plugins/alerting/common/rule.ts

```diff
@@ -89,8 +90,23 @@ export interface RuleAction {
   };
 }
 
-export interface RuleAggregations {
-  alertExecutionStatus: { [status: string]: number };
+type IndexType = Record<string, unknown>;
+
+export interface AggregateOptions extends IndexType {
```

Comment:
I'm just curious if you want to be restrictive but `AggregateOptions` extends `Record<string, unknown>` which means I can define any possible field here so it will be passed to `unsecuredSavedObjectsClient.find()`. Can it have some security impact?

Reply:
Sounds good, a lot of this stuff is legacy so I will just remove it, we can bring it back if there's a good reason to.